Package.swift

```diff
@@ -24,14 +24,18 @@ let package = Package(
         .library(name: "SpeziLLMOpenAI", targets: ["SpeziLLMOpenAI"])
     ],
     dependencies: [
-        .package(url: "https://github.com/MacPaw/OpenAI", .upToNextMinor(from: "0.2.5")),
+        .package(
+            url: "https://github.com/MacPaw/OpenAI",
+            revision: "35afc9a6ee127b8f22a85a31aec2036a987478af"    // No new release from MacPaw, use commit on main until release is tagged
+        ),
         .package(url: "https://github.com/StanfordBDHG/llama.cpp", .upToNextMinor(from: "0.1.8")),
         .package(url: "https://github.com/StanfordSpezi/Spezi", from: "1.1.0"),
         .package(url: "https://github.com/StanfordSpezi/SpeziStorage", from: "1.0.0"),
         .package(url: "https://github.com/StanfordSpezi/SpeziOnboarding", from: "1.0.0"),
         .package(url: "https://github.com/StanfordSpezi/SpeziSpeech", from: "1.0.0"),
-        .package(url: "https://github.com/StanfordSpezi/SpeziChat", .upToNextMinor(from: "0.1.4")),
-        .package(url: "https://github.com/StanfordSpezi/SpeziViews", from: "1.0.0")
+        .package(url: "https://github.com/StanfordSpezi/SpeziChat", .upToNextMinor(from: "0.1.5")),
+        .package(url: "https://github.com/StanfordSpezi/SpeziViews", from: "1.0.0"),
+        .package(url: "https://github.com/groue/Semaphore.git", exact: "0.0.8")
```

Comment:
Might not be that urgent here but we should maybe create an issue to see that we can pull @Supereg's `AsyncSemaphore` in Spezi Foundation and utilize it here?

https://github.com/StanfordSpezi/SpeziBluetooth/pull/10/files#diff-d615550338a5a3f773d26989249b8da9396076affa5255da8923eef20716ab63

Reply:
Good call, thx for the hint!